pkg/webui/lib/errors/utils.js

```diff
@@ -42,11 +42,10 @@ export const hasValidDetails = object =>
  * @returns {boolean} `true` if `error` is a well known backend error object.
  */
 export const isBackend = error =>
-  Boolean(error) &&
-  typeof error === 'object' &&
+  isPlainObject(error) &&
   !('id' in error) &&
   error.message &&
-  (error.code || error.grpc_code)
+  (typeof error.code === 'number' || typeof error.grpc_code === 'number')
```

Comment:
Backend errors always have a `code` or `grpc_code` with a number. Eg timout errors had the format that you can observe in the `utils_test.js` which would hence formerly pass through the `isBackend()` check.